zerver/migrations/0272_preregistrationuser_user_avatar_url.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.10 on 2020-03-25 19:47
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0271_huddle_set_recipient_column_values'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='preregistrationuser',
+            name='user_avatar_url',
+            field=models.CharField(max_length=900, null=True),
```

Comment:
For a variable-length string, use TextField; this will allocate 900 bytes of memory to each object, which is no good.  You'll want to change it in `models.py` and then delete and regenerate this migration.